src/test/java/org/jitsi/meet/test/mobile/base/MobileParticipantFactory.java

```diff
@@ -38,21 +43,71 @@ public MobileParticipantFactory(Properties config)
         super(config);
     }
 
+    /**
+     * Include global properties specific to mobile.
+     * {@inheritDoc}
+     */
+    @Override
+    protected List<String> getGlobalConfigKeys()
+    {
+        List<String> systemGlobalKeys = new LinkedList<>();
+
+        // If at any point Android or iOS will require to handle global
+        // properties they will have to be included here as well. The reason why
+        // it's done this way is that the ParticipantOptions type is not known
+        // until config properties are loaded and merged with the explicit
+        // options.
+        systemGlobalKeys.addAll(
+            MobileParticipantOptions.getSystemGlobalPropNames());
+
+        return systemGlobalKeys;
+    }
+
+    /**
+     * {@inheritDoc}
+     */
     @Override
-    public Participant<? extends WebDriver> createParticipant(
-        String configPrefix,
-        ParticipantOptions options)
+    public Participant<? extends WebDriver> doCreateParticipant(
```

Comment:
And this is why I don't like force pushing :)

Reply:
what about this one ?

Reply:
To clarify: this was fixed to "protected" in the previous version of the PR (6d7b4a74e6e8d0a5f30809b3b56bd03b6b800cdb), and it's not easy to spot the difference in the gh interface.

Reply:
ok, got it

Reply:
good job on catching this one :)